le::ops, 0);
		testAssert("fail gracefully on bad table", !face);
	}
	catch (std: